src/i18n/locales/en/translation.json

```diff
@@ -453,6 +454,47 @@
             "title": "Your Secret Key"
           }
         },
+        "liquid": {
+          "title": "Liquid",
+          "hint": "have to fill this.",
+          "advanced_settings": {
+            "label": "Advanced Settings",
```

Comment:
most or all this is the same as Nostr right? maybe we should consider using a variable {coinType} or something instead, since you made a shared component?

Reply:
yeah I was thinking to make one but missed a few things which were to be shared as there were lot of files haha, thanks for reminding 🙌